x-pack/plugins/enterprise_search/public/applications/workplace_search/views/setup_guide/setup_guide.tsx

```diff
@@ -55,7 +55,10 @@ export const SetupGuide: React.FC = () => {
       </EuiTitle>
       <EuiSpacer size="l" />
       <EuiButton target="_blank" fill href={GETTING_STARTED_LINK_URL} iconType="popout">
-        Get started with Workplace Search
+        <FormattedMessage
```

Comment:
Why was `FormattedMessage` used here instead of a constant?

Reply:
I went for consistency within the file here. FormattedMessage is also used below on lines 66-69.

I think it's worth it to revisit the entire i18n architecture within our app. Let's discuss it separately? Not postponing to "sometimes in the future", just want to get to alignment first. Once we decide on the architecture, happy to update the entire app to match it.